plugins/deployed-labeler/main.go

```diff
@@ -140,6 +197,15 @@ func (s *server) handleMarkDeployedPRs(ctx context.Context, commitSHA, team stri
 	return s.updatePullRequests(prs, team)
 }
 
+func (s *server) handleGetUnmarkedPRs(ctx context.Context, commitSHA, team string) (labeledPRs, unlabeledPRs []string, errors []error) {
+	prs, err := s.getMergedPRs(ctx, commitSHA)
+	if err != nil {
+		return nil, nil, []error{err}
+	}
```

Comment:
```suggestion
func (s *server) handleGetUnmarkedPRs(ctx context.Context, commitSHA, team string) (labeledPRs, unlabeledPRs []string, errors error) {
	prs, err := s.getMergedPRs(ctx, commitSHA)
	if err != nil {
		return nil, nil, err
	}
```

Why use an array if you'll never return more than one error?

Reply:
> Why use an array if you'll never return more than one error?

This seems to have done as `s.findTeamPullRequests` has a return type of `(labeled, unlabeled []string, errs []error)`. We are just passing those over for the caller to use as needed.

Reply:
Right. Updated it to use a single error.